### PR TITLE
Tix working branch

### PR DESCRIPTION
**Pull request by Tixxx**

### CODE REVIEW

**Comment by saldana** [comment]
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C12%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703120726503212-215%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C14%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703141755154438-215%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting.sln](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting.sln) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/DLLTrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/DLLTrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Microsoft.Practices.Unity.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Microsoft.Practices.Unity.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Properties/AssemblyInfo.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Properties/AssemblyInfo.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterBasedEventChannel.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterBasedEventChannel.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/CLITrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/CLITrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/ManagedTrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/ManagedTrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/ManagedTrouterClientCommon.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/ManagedTrouterClientCommon.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Microsoft.Skype.Calling.Framework.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Microsoft.Skype.Calling.Framework.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Microsoft.Skype.Calling.ServiceAgents.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Microsoft.Skype.Calling.ServiceAgents.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Newtonsoft.Json.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Newtonsoft.Json.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Skype.Common.Azure.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Skype.Common.Azure.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/System.Net.Http.Formatting.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/System.Net.Http.Formatting.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/System.Web.Http.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/System.Web.Http.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/TrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/TrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/packages.config](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/packages.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/UpgradeLog.htm](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/UpgradeLog.htm) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C14%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703142108378375-215%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj [hunk 1]
```diff
@@ -0,0 +1,149 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj]: You are referencing dlls from TrouterBasedEventChannel project here. You don't need to add those dlls to MessagingAfterJoinMeeting directory.

File: Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs [hunk 2]
```diff
@@ -1,26 +1,73 @@
-﻿using Microsoft.SfB.PlatformService.SDK.Common;
+using Microsoft.SfB.PlatformService.SDK.Common;
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs]: ConsoleLogger should not log to file. If you want, create a new class FileLogger and use that in samples.

File: Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs [hunk 3]
```diff
@@ -94,7 +94,6 @@ internal class TrustedJoinMeeting
             await invitation.WaitForInviteCompleteAsync().ConfigureAwait(false);
 
             invitation.RelatedConversation.HandleParticipantChange += Conversation_HandleParticipantChange;
-
             WriteToConsoleInColor("Showing roaster udpates for 5 minutes for meeting : " + adhocMeeting.JoinUrl);
 
             // Wait 5 minutes before exiting.
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs]: Please remove UpgradeLog.htm from tracking.

File: Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config [hunk 4]
```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <startup> 
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
+  </startup>
+  <appSettings>
+    <add key="AAD_ClientId" value="418fbf0c-d180-4cd9-8d13-7d0e9babbb9e" />
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config]: We don't want to publish values, please replace these with stubs/placeholders like we have for other projects.

File: Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+            var invitation = await adhocMeeting.JoinAdhocMeeting(loggingContext, null).ConfigureAwait(false);
+
+            // Wait for the join to complete
+            await invitation.WaitForInviteCompleteAsync().ConfigureAwait(false);
+
+            invitation.RelatedConversation.HandleParticipantChange += Conversation_HandleParticipantChange;
+
+            WriteToConsoleInColor("Showing roaster udpates for 1 minute for meeting : " + adhocMeeting.JoinUrl);
+
+            // Wait 5 minutes before exiting.
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs]: Remove this section

File: Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+            // Get all the events related to join meeting through Trouter's uri
+            platformSettings.SetCustomizedCallbackurl(new Uri(EventChannel.CallbackUri));
+
+            // Start joining the meeting
+            var invitation = await adhocMeeting.JoinAdhocMeeting(loggingContext, null).ConfigureAwait(false);
+
+            // Wait for the join to complete
+            await invitation.WaitForInviteCompleteAsync().ConfigureAwait(false);
+
+            invitation.RelatedConversation.HandleParticipantChange += Conversation_HandleParticipantChange;
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs]: We already have samples for this. You can remove this and related code from here to keep this sample project focused on IM and messages.

File: Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+            var modalities = invitation.RelatedConversation.ActiveModalities;
+            WriteToConsoleInColor("Active modality is : ");
+            foreach (var modality in modalities)
+            {
+                WriteToConsoleInColor(modality.ToString() + " ");
+            }
+
+            await imCall.SendMessageAsync("Hello World.", loggingContext).ConfigureAwait(false);
+
+            await Task.Delay(TimeSpan.FromSeconds(180)).ConfigureAwait(false);
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs]: Minor: TimeSpan.FromMinutes

File: Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+            }
+
+            var modalities = invitation.RelatedConversation.ActiveModalities;
+            WriteToConsoleInColor("Active modality is : ");
+            foreach (var modality in modalities)
+            {
+                WriteToConsoleInColor(modality.ToString() + " ");
+            }
+
+            await imCall.SendMessageAsync("Hello World.", loggingContext).ConfigureAwait(false);
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs]: Write something on console to indicate success/failure

File: Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup>
+    <StartupObject>MessagingAfterJoinMeeting.Program</StartupObject>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System.Web.Extensions" />
+    <Reference Include="TrouterBasedEventChannel">
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj]: Please reference TrouterBasedEventChannel.csproj file, not the dll

File: Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterBasedEventChannel.cs [hunk 10]
```diff
@@ -0,0 +1,180 @@
+﻿using System;
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterBasedEventChannel.cs]: You are referencing TrouterBasedEventChannel.csproj, you don't need this file/class

File: Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj (hunk 9 above)
Comment (Tixxx) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj]: we probably need to remove it in Trusted meeting join as well

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting.sln](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting.sln) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/DLLTrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/DLLTrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Microsoft.Practices.Unity.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Microsoft.Practices.Unity.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Properties/AssemblyInfo.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Properties/AssemblyInfo.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterBasedEventChannel.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterBasedEventChannel.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/CLITrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/CLITrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/ManagedTrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/ManagedTrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/ManagedTrouterClientCommon.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/ManagedTrouterClientCommon.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Microsoft.Skype.Calling.Framework.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Microsoft.Skype.Calling.Framework.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Microsoft.Skype.Calling.ServiceAgents.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Microsoft.Skype.Calling.ServiceAgents.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Newtonsoft.Json.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Newtonsoft.Json.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Skype.Common.Azure.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/Skype.Common.Azure.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/System.Net.Http.Formatting.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/System.Net.Http.Formatting.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/System.Web.Http.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/System.Web.Http.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/TrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterClient/TrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/packages.config](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/packages.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C15%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703150533041126-215%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting.sln](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting.sln) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/DLLTrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/DLLTrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Microsoft.Practices.Unity.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Microsoft.Practices.Unity.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Properties/AssemblyInfo.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Properties/AssemblyInfo.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterBasedEventChannel.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterBasedEventChannel.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/packages.config](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/packages.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/TrustedJoinMeeting.csproj](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/TrustedJoinMeeting.csproj) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C15%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703150537056157-215%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting.sln](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting.sln) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/DLLTrouterClient.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/DLLTrouterClient.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/MessagingAfterJoinMeeting.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Microsoft.Practices.Unity.dll](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Microsoft.Practices.Unity.dll) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Program.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Properties/AssemblyInfo.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/Properties/AssemblyInfo.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/packages.config](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/packages.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/ConsoleLogger.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/Program.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/TrustedJoinMeeting.csproj](https://github.com/Tixxx/skype-docs/blob/tixWorkingBranch/Skype/Trusted-Application-API/samples/QuickStartSamples/TrustedJoinMeeting/TrustedJoinMeeting.csproj) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C16%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703161706397570-215%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**